### ISSUE TITLE
[Feature] Windows Support

### ISSUE THREAD

**Issue by tracker1** [Phate6660/nixinfo#8]
Similar to MacOS Support #5, it would be nice if Windows was a supported target,  It may be possible to reference [winfetch](https://github.com/kiedtl/winfetch) for some inspiration.

This should probably include Win-Get and Chocolatey package counts.

**Comment by Phate6660** [comment]
Trust me, this is something I already plan lol.
My main issue is that Windows is a confusing pain (I say this on a Windows laptop).
Currently I've been looking through Microsoft's new Windows API crate for Rust, since it seems like I'll need to use the Windows API to get any sort of info I want.

**Comment by tracker1** [comment]
Yeah, no real reference for what it will take... I'm mostly on Linux personally and Windows for work... but given the target and usage, would be nice to have info on at least the big three platforms.
